rfcs/0000-policy_target_scope.md

```diff
@@ -0,0 +1,285 @@
+
+```yaml
+enabled: false
+rps: 500
+```
+
+Which would effectively disable all rate limiting traffic to the `foo-route` and ultimately the `foo`
+`Backend`.
+
+Using another example, let's consider a single `HTTPRoute` with two `HTTPRouteRule`s:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: bar-route
+spec:
+  parentRefs:
+  - name: example-gateway
+  hostnames:
+  - "bar.example.com"
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /login
+    backendRefs:
+    - name: bar-svc
+      port: 8081
+  - matches:
+    - headers:
+      - type: Exact
+        name: env
+        value: production
+    backendRefs:
+    - name: bar-svc
+      port: 8080
+```
+
+It is important to consider that only _one_ rule will ever be match for any given requests. It makes for a simpler model to have only one "Policy" to also only ever being match, aligning the both behaviors. To apply a policy on the first rule only, the user needs to duplicate that match on the policy targeting `bar-route`:
+
+```yaml
+kind: FakeRLPolicy
+spec:
+  default:
+    rps: 50
+    rules:
+    - matches:
+      - path:
+          type: PathPrefix
+          value: /login
+  targetRef:
+    kind: HTTPRoute
+    name: foo-route
+```
+
+Hereby limiting the amount of hits to our `/login` on port `8081` to a 50 requests per second. That limit _will not_
+apply to other matches. So that Limitador will be configured by the Kuadrant controller to not rate limit traffic going
+to port `8080`.
+
+Merging only occurs at the `HTTPRoute` or within individual `HTTPRouteRule` within a route. The merging strategy will
+resolve the individual `HTTPRouteRule` all the policies boil down to and merge according to the Gateway API Policy
+Attachment hierarchy. Targeting a complete `HTTPRoute` is the equivalent of targeting each indidividual `HTTPRouteRule`s
+that composes the route.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+So there are a couple of things to consider here:
+
+- status reporting on "not yet" existing `HTTPRouteRule`, e.g. a Policy declares a set of `matches` that don't (yet?)
+  exist. The policy must _not_ be enforced... yet. But could eventually. This needs proper reporting. 
+
+This is the technical portion of the RFC. Explain the design in sufficient detail that:
+
+- Its interaction with other features is clear.
+- It is reasonably clear how the feature would be implemented.
+- How error would be reported to the users.
+- Corner cases are dissected by example.
+
+The section should return to the examples given in the previous section, and explain more fully how
+the detailed proposal makes those examples work.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+- The use cases this approach doesn't address… 
+- Not very DRY... the need to keep `HTTPRouteRule`s in sync between the Policy and the `HTTPRoute` is suboptimal.
```

Comment:
This is probably the most discouraging point, but the good thing is we could mitigate a bit with status reporting and ux tooling.